lib/src/kbasesearchengine/main/DecorateWithNarrativeInfo.java

```diff
@@ -0,0 +1,77 @@
+package kbasesearchengine.main;
+
+import kbasesearchengine.*;
+import kbasesearchengine.ObjectData;
+import kbasesearchengine.common.GUID;
+import kbasesearchengine.events.exceptions.IndexingException;
+import kbasesearchengine.events.exceptions.RetriableIndexingException;
+import kbasesearchengine.events.handler.CloneableWorkspaceClientImpl;
+import kbasesearchengine.events.handler.EventHandler;
+import kbasesearchengine.events.handler.WorkspaceEventHandler;
+import workspace.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by umaganapathy on 1/26/18.
+ */
+public class DecorateWithNarrativeInfo extends ObjectDecorator {
```

Comment:
Doesn't appear that this class is used anywhere?

Reply:
Added the call to KBaseSearchEngineServer. Used in ObjectDecoratorTest.